cfpb/consumerfinance.gov
# Add tour to ccdb

Adding tour to CCDB

## Additions

- Tour module for CCDB

## Removals

-


## Changes

-


## How to test this PR

1. run local code
2. visit https://localhost:8000/data-research/consumer-complaints/search/?chartType=line&dateInterval=Month&dateRange=3y&date_received_max=2022-09-22&date_received_min=2019-09-22&lens=Overview&searchField=all&tab=Trends
3. Click Take a Tour button in bottom right
4. confirm tour appears

## Screenshots
<img width="190" alt="Screen Shot 2022-09-22 at 10 35 56 AM" src="https://user-images.githubusercontent.com/1038306/191776372-cfa16bb4-5a87-4592-ac71-f85d23a18c0a.png">
<img width="1311" alt="Screen Shot 2022-09-22 at 10 36 05 AM" src="https://user-images.githubusercontent.com/1038306/191776374-5cc59e5e-a3b8-4dbd-a658-ac600e894548.png">



## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
